packages/address-service-client/README.md

```diff
@@ -0,0 +1,12 @@
+Address service client
+==
+
+The client for condo address service
+
+***
+
+## How to use:
+
+```
+const client = new AddressServiceClient(...)
```

Comment:
it's better to have some inner logic that will decide what type of client we need Fake or NonFake

Reply:
I've fix this example